File: src/main/java/com/procurement/submission/model/dto/Document.java
package com.procurement.submission.model.dto;

import java.util.List;
import lombok.Data;

@Data
public class Document {
    public String id;
    public String documentType;
    public String title;
    public String description;
    public String url;
    public String datePublished;
    public String dateModified;
    public String format;
    public String language;
    public List<String> relatedLots;
}

File: src/main/kotlin/com/procurement/submission/model/dto/ocds/Document.kt
package com.procurement.submission.model.dto.ocds

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude
import com.procurement.submission.domain.model.enums.DocumentType

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Document @JsonCreator constructor(

    val id: String,

    val documentType: DocumentType,

    @JsonInclude(JsonInclude.Include.NON_NULL)
    var title: String?,

    @JsonInclude(JsonInclude.Include.NON_NULL)
    var description: String?,

    @JsonInclude(JsonInclude.Include.NON_EMPTY)
    var relatedLots: List<String>?
)